### DISCUSSION TITLE
How to use SkipLocked and ordering in v0.0.29

### DISCUSSION THREAD

**Discussion by dmakushin** [stephenafamo/bob#316] [Q&A] [answered]
How to use additional query parameters in v0.29.0? 
In example below I faced with errors for both `SkipLocked` and `Acs` which says that returns types which don't implement `Apply` correctly anymore.
```
	queryMods := []bob.Mod[*dialect.SelectQuery]{
		model.SelectWhere.QueueItems.ObjectType.EQ(string(objType)),
		sm.ForUpdate(model.TableNames.QueueItems).SkipLocked(),
		sm.OrderBy(model.QueueItemColumns.CreatedAt).Asc(),
	}
```

Example of error
```
```Cannot use 'sm.ForUpdate(model.TableNames.QueueItems).SkipLocked()' (type LockChain[Q]) as the type bob.Mod[*dialect.SelectQuery]Type does not implement 'bob.Mod[*dialect.SelectQuery]'need the method: Apply(*dialect.SelectQuery)have the method: Apply(q Q)```
```

**Comment by stephenafamo** [comment]
My guess is that the `sm` package you have imported is for a different dialect.

Because the packages are named the same, the error can be confusing.

**Comment by dmakushin** [comment]
This code worked before update to v0.29.0 and nothing has changed in imports. I use psql dialect.

**Reply by stephenafamo** [reply, marked as answer]
Added a test in 458d070 which shows it is working. Not sure what the issue is.
